assets/i18n/en.toml

```diff
@@ -45,28 +45,28 @@ savingsTotalEnergy = "{total} kWh charged"
 tabTitle = "Live community"
 power = "Charging power"
 powerSub1 = "{activeClients} of {totalClients} participants"
-powerSub2 = "are charging right now"
+powerSub2 = "charging…"
 greenShare = "Solar share"
-greenShareSub1 = "power is covered by"
-greenShareSub2 = "PV & battery storage"
-greenEnergy = "Solar energy"
+greenShareSub1 = "power derived from"
+greenShareSub2 = "solar, and battery storage"
+greenEnergy = "Solar"
```

Comment:
A bit 50/50 on this one.

Reply:
Should it not be => greenShareSub2 = "solar and battery storage"
removing "," ;-)

Reply:
It is two different clauses. Nothing is really stored in the solar.